pkgs/top-level/ocaml-packages.nix

```diff
@@ -231,6 +231,8 @@ let
 
     earley = callPackage ../development/ocaml-modules/earley { };
 
+    earlybird = callPackage ../development/ocaml-modules/ocamlearlybird { };
```

Comment:
What’s the name of this package? “earlybird” or “ocamlearlybird”? There is contradictory information on this line.

Reply:
I need a hint here, because the source repository is named `ocamlearlybird`, and the opam package is named `earlybird`. What do you suggest to keep it consistent on nixpkgs?

Reply:
The “ocaml” prefix looks redundant here; had I to choose, I’d go for “earlybird”.